roles/tomcat9/molecule/default/converge.yml

```diff
@@ -0,0 +1,15 @@
+---
+- name: Converge
+  hosts: all
+  vars:
+    - running_on_server: false
+  become: true
+  pre_tasks:
+    - name: update cache
+      ansible.builtin.apt:
+        update_cache: true
+        cache_valid_time: 600
+  tasks:
+    - name: "Include tomcat8"
+      ansible.builtin.include_role:
+        name: tomcat8
```

Comment:
This one would be particularly challenging if we continue to have a role called `tomcat8`, since then the tomcat9 tests would actually be running the tomcat8 tasks . . .